frontend/src/main/web/src/containers/App.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+// import a11y from 'react-a11y'
```

Comment:
Is this here as a reminder, or just forgotten? If it is a reminder, say `TODO` there so it is clear.